rohmu/inotify.py

```diff
@@ -42,7 +48,7 @@ class InotifyEvent(ctypes.Structure):
 IN_NONBLOCK = 0x00004000
 
 
-def parse_inotify_buffer(event_buffer):
+def parse_inotify_buffer(event_buffer: bytes) -> Iterator[tuple[int, int, int, bytes]]:
```

Comment:
Some docstring with output explanation or better a named tuple could be useful here